src/explore-education-statistics-frontend/src/modules/methodologies/MethodologyIndexPage.tsx

```diff
@@ -12,6 +12,8 @@ import PageSearchFormWithAnalytics from '@frontend/components/PageSearchFormWith
 import orderBy from 'lodash/orderBy';
 import { GetServerSideProps, NextPage } from 'next';
 import React from 'react';
+import { MethodologySummary } from '@common/services/types/methodology';
+import { uniqWith } from 'lodash';
```

Comment:
When using Lodash, we need use the fully qualified import i.e. 

```ts
import uniqWith from 'lodash/uniqWith';
```

Reply:
There were a few other instances in the codebase where we did this, so I changed those as well.